src/app/planner/card/card.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="card" >
+		<div class="card-info">
+		<img class="meal-img" src='{{url}}' alt="meal">
+		<h2><ng-content></ng-content></h2>
+		</div>
+	  <div class="card-portion">
+		<h3>{{portion}}</h3>
+			<button id="remove-button" (click)="deleteMeals(id)"><h3>X</h3></button>
```

Comment:
The x symbol should not be a heading tag. The best would be to use an image or a svg. In this case it will do to just use a span. 
But a h3 actually signifies that it is an important heading, which it is not.